### DISCUSSION TITLE
Computer vision on publish?

### DISCUSSION THREAD

**Discussion by rcloran** [rcloran/lr-inaturalist-publish#3] [Ideas]
I think it would be neat if you could use iNaturalist's computer vision system to identify during upload. I find the keywording system in Lightroom hard to use to apply the keywords I want so that observations get the correct initial identification. WDYT?

**Comment by dominictinner** [comment]
That sounds neat! Before your post, I imagined a page on inaturalist.org, where you could see all your new observations that haven't been identified. Now I usually open all new observations in new tabs and work through them. Is the identification module used on the website accessible so you could implement it into Lightroom?

**Reply by rcloran** [reply]
> Now I usually open all new observations in new tabs and work through them

I do the same. I've thought it might be useful in the interim to automatically open a browser tab for each observation that's created. Do you think that might be useful for you?

**Comment by rcloran** [comment]
It would have to be a custom UI within Lightroom, but the vision functionality of obtaining a list of possible matches based on an uploaded photo is available by API. However, it's not documented in the API, so I'm not sure whether it's intended to be used outside of the official apps.
